app/views/NotificationPreferencesView/index.js

```diff
@@ -0,0 +1,263 @@
+	}],
+	emailNotifications: [{
+		label: I18n.t('Default'), value: 'default'
+	}, {
+		label: I18n.t('All Messages'), value: 'all'
+	}, {
+		label: I18n.t('Mentions'), value: 'mentions'
+	}, {
+		label: I18n.t('Nothing'), value: 'nothing'
+	}],
+	desktopNotificationDuration: [{
+		label: I18n.t('Default'), value: 0
+	}, {
+		label: I18n.t('Seconds', { second: 1 }), value: 1
+	}, {
+		label: I18n.t('Seconds', { second: 2 }), value: 2
+	}, {
+		label: I18n.t('Seconds', { second: 3 }), value: 3
+	}, {
+		label: I18n.t('Seconds', { second: 4 }), value: 4
+	}, {
+		label: I18n.t('Seconds', { second: 5 }), value: 5
+	}],
+	audioNotificationValue: [{
+		label: 'None', value: 'none None'
+	}, {
+		label: I18n.t('Default'), value: '0 Default'
+	}, {
+		label: 'Beep', value: 'beep Beep'
+	}, {
+		label: 'Ding', value: 'ding Ding'
+	}, {
+		label: 'Chelle', value: 'chelle Chelle'
+	}, {
+		label: 'Droplet', value: 'droplet Droplet'
+	}, {
+		label: 'Highbell', value: 'highbell Highbell'
+	}, {
+		label: 'Seasons', value: 'seasons Seasons'
+	}]
+};
+
+export default class NotificationPreferencesView extends React.Component {
+	static navigationOptions = () => ({
+		title: I18n.t('Notification_Preferences')
+	})
+
+	static propTypes = {
+		navigation: PropTypes.object
+	}
+
+	constructor(props) {
+		super(props);
+		this.rid = props.navigation.getParam('rid');
+		this.rooms = database.objects('subscriptions').filtered('rid = $0', this.rid);
+		this.state = {
+			room: JSON.parse(JSON.stringify(this.rooms[0] || {}))
+		};
+	}
+
+	onValueChangeSwitch = async(key, value) => {
+		const { room: newRoom } = this.state;
+		newRoom[key] = value;
+		this.setState({ room: newRoom });
+		const params = {
+			[key]: value ? '1' : '0'
+		};
+		try {
+			await RocketChat.saveNotificationSettings(this.rid, params);
+		} catch (err) {
+			log('err_save_notification_settings', err);
+		}
+	}
+
+	onValueChangePicker = async(key, value) => {
+		const { room: newRoom } = this.state;
+		newRoom[key] = value;
+		this.setState({ room: newRoom });
+		const params = {
+			[key]: value.toString()
+		};
+		try {
+			await RocketChat.saveNotificationSettings(this.rid, params);
+		} catch (err) {
+			log('err_save_notification_settings', err);
+		}
+	}
+
+	renderPicker = (key) => {
+		const { room } = this.state;
+		return (
+			<RNPickerSelect
+				testID={key}
+				style={{ viewContainer: styles.viewContainer }}
+				value={room[key]}
+				textInputProps={{ style: styles.pickerText }}
+				useNativeAndroidPickerStyle={false}
+				placeholder={{}}
+				onValueChange={value => this.onValueChangePicker(key, value)}
+				items={OPTIONS[key]}
+			/>
+		);
+	}
+
+	renderSwitch = (key) => {
+		const { room } = this.state;
+		return (
+			<Switch
+				value={!room[key]}
+				testID={key}
+				trackColor={SWITCH_TRACK_COLOR}
+				onValueChange={value => this.onValueChangeSwitch(key, !value)}
+			/>
+		);
+	}
+
+	render() {
+		const { room } = this.state;
+		return (
```

Comment:
I'm missing SafeAreaView notch control (take a look at bottom cutting):
<img src="https://user-images.githubusercontent.com/804994/63286259-036b0280-c28e-11e9-871c-eb8d8521e602.png" width="350" />

Reply:
its not coming for me